lib/locales/es.ts

```diff
@@ -224,6 +224,7 @@ export const locale: Locale = {
     benedict_of_aniane_abbot: 'San Benito de Aniano, Abad',
     benedict_of_jesus_valdivielso_saez_religious: 'San Héctor Valdivielso Sáez, mártir',
     benedict_of_nursia_abbot: 'San Benito, abad, patrono de Europa,',
```

Comment:
There should be no commma at the end of the value. And a patronage title was left here.

```suggestion
    benedict_of_nursia_abbot: 'San Benito, abad',  // src: https://liturgiapapal.org/attachments/article/1093/PROPIO%20DE%20LOS%20SANTOS.pdf#page=9
```

Reply:
resolved in #622